app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,7 @@
+a {
+  border: none;
```

Comment:
__[scss_lint]__
* [BorderZero] `border: 0` is preferred over `border: none`

<!-- Comment by [SideCI](https://sideci.com) -->